test/ssc_test/cmod_singleowner_test.cpp

```diff
@@ -102,7 +102,7 @@ TEST_F(CmodSingleOwnerTest, ETES) {
 
     Test("singleowner", file_inputs, file_outputs, compare_number_variables, compare_array_variables);
 }
-
+/*
 TEST_F(CmodSingleOwnerTest, PV) {
```

Comment:
As PV-SingleOwner is likely one of the most modeled technologies, I might consider keeping this one and commenting out others. Although I see PV Battery is still being run, so there may be enough coverage already. Same thought for PVWatts-SingleOwner

Reply:
The ETES test was kept because it uses an additional o and m value that the others single owner tests do not. Same with the battery and fuel cell. The other tests were selected based on inputs to single owner compute module being changed. The PV and PVwatts single owner tests were added back (no coverage change and only an additional 40ms added)